src/Sarif/HashUtilities.cs

```diff
@@ -31,15 +31,14 @@ internal static IFileSystem FileSystem
 
         public static IDictionary<string, HashData> MultithreadedComputeTargetFileHashes(IEnumerable<string> analysisTargets, bool suppressConsoleOutput = false)
         {
-            if (analysisTargets == null) { return null; }
+            var fileToHashDataMap = new ConcurrentDictionary<string, HashData>();
+            if (analysisTargets == null) { return fileToHashDataMap; }
```

Comment:
This will prevent a NullReferenceException when we use the reerence variable from the SarifLogger